01_loaddata.R

```diff
@@ -1,25 +1,78 @@
 
-# methdology for Redknot conversion to R
+# Redknot Methodology Conversion to R
 
+# Load R Packages
 library(fs)
 library(dplyr)
 library(readr)
 library(sf)
-library(terra) #testing line
-library(lidR) # cam test
+library(terra)
 
-# 1) read in csv from 01_inputs folder
+## DATA PRE-PROCESSING ##
+# Read CSV Data
+birds <- read_csv(path("01_inputs", "DS 2 _P7165_Con_213835_250122134729.csv"),name_repair = "unique",locale=locale(encoding="latin1"))
 
-bd <- read_csv(path("01_inputs", "DS 2 _P7165_Con_213835_250122134729.csv"),name_repair = "unique",locale=locale(encoding="latin1"))
+# Convert to SF Object
+birds_sf <- st_as_sf(birds, coords = c("Longitude", "Latitude"), crs = 4326)
```

Comment:
```suggestion
birds_sf <- st_as_sf(birds, coords = c("Longitude", "Latitude"), crs = 4326, remove = FALSE)
```
I think we can keep the lat long columns here and use at a later stage in the process